man/get_files_content.Rd

```diff
@@ -41,8 +55,20 @@ git hosts).
     token = Sys.getenv("GITLAB_PAT_PUBLIC"),
     orgs = "mbtests"
   )
- get_files(
+ get_files_content(
+   gitstats_obj = my_gitstats,
+   file_path = c("LICENSE", "DESCRIPTION")
+ )
+
+ # example with files structure
+ files_structure <- get_files_structure(
```

Comment:
I was testing example reproducibility and stumbled on the ouput below. The final result in the table is correct - the message though is a result of not finding any files on GitLab site. Maybe we can alter the message, as it sounds like there's some error which may be misleading: 

```
> # get_files_content() will make use of pulled earlier files structure
> files_content <- get_files_content(
+   gitstats_obj = my_gitstats
+ )
ℹ Parameters changed, I will pull data from API.
ℹ I will make use of files structure stored in GitStats.
ℹ [Host:GitHub][Engine:GraphQl][Scope:r-world-devs] Pulling files from files structure...
ℹ [Host:GitLab][Engine:GraphQl][Scope:mbtests] Pulling files content: []...
! Variable $file_paths of type [String!]! was provided invalid value132Expected value to not be null
✖ Empty response.
```